Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/LocalData.kt

```diff
@@ -750,4 +750,24 @@ object LocalData {
      ****************************************************/
 
     fun getSharedPreferenceInstance(): SharedPreferences = globalEncryptedSharedPreferencesInstance
+
+    /****************************************************
+     * INTEROPERABILITY
```

Comment:
Proposal: Instead of having these section Comments, we could rather use inner classes/objects to make this more separation more obvious in code. E.g. `LocalData.InterOp.saveInteroperabilityUsed()`

Reply:
Good idea! @d4rken what do you think about this idea?

Reply:
:+1:
I like @ralfgehrer's propsal. I'm a huge fan of descriptive code, and also of nested classes. I also like @chris-cwa's idea, component specific settings would be awesome too.

We'll probably need a bit of glue code to be able to refactor this without touching everything that is related. Should we do this in an extra PR?

Reply:
If we can decide how we should handle it in future, we can also start with this PR. But if we're not 100% sure how exactly we should do it I think an own PR for this would be better

Reply:
Agreed. Dedicated PR is fine. Would you mind creating a ticket for it?